File: chat_hub/kernel/config/security_config.py
import os


class SecurityConfig:
    def __init__(self):
        self.private_token = os.getenv('PRIVATE_TOKEN', '')
        self.public_key = os.getenv('PUBLIC_KEY', '')
        self.private_key = os.getenv('PRIVATE_KEY', '')
        self.sse_public_key = os.getenv('SSE_PUBLIC_KEY', '')
        self.sse_private_key = os.getenv('SSE_PRIVATE_KEY', '')


security_config = SecurityConfig()

File: person_task_manager/new_server/kernel/configs/security_config.go
package configs

import (
	"os"

	"github.com/joho/godotenv"
)

type SecurityConfig struct {
	PublicKey     string
	PrivateKey    string
	PrivateToken  string
	SSEPublicKey  string
	SSEPrivateKey string
}

func (in *SecurityConfig) LoadSecurityEnv() (SecurityConfig, error) {
	err := godotenv.Load(".env")
	if err != nil {
		return SecurityConfig{}, err
	}

	publicKey := os.Getenv("RSA_SIGNATURE.PUBLIC_KEY")
	privateKey := os.Getenv("RSA_SIGNATURE.PRIVATE_KEY")
	privateToken := os.Getenv("RSA_SIGNATURE.PRIVATE_TOKEN")
	ssePublicKey := os.Getenv("RSA_SIGNATURE.SSE_PUBLIC_KEY")
	ssePrivateKey := os.Getenv("RSA_SIGNATURE.SSE_PRIVATE_KEY")

	securityConfig := SecurityConfig{
		PublicKey:     publicKey,
		PrivateKey:    privateKey,
		PrivateToken:  privateToken,
		SSEPublicKey:  ssePublicKey,
		SSEPrivateKey: ssePrivateKey,
	}
	return securityConfig, nil
}